app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListViewHolderFactory.kt

```diff
@@ -439,7 +447,7 @@ fun ElapsedInterval.format(resources: Resources) = when (unit) {
     else -> resources.getString(R.string.tx_list_ago_min, value)
 }
 
-fun BlockiesImageView.loadKnownAddressLogo(logoUri: String?, address: Solidity.Address, target: Target) {
+fun KnownAddressLogoView.loadKnownAddressLogo(logoUri: String?, address: Solidity.Address, target: Target) {
```

Comment:
not critical: target could be set to this by default

Reply:
Unfortunately `this` would not be recognized as a `Target` in an extension function. So I made it an instance method. 

Done in: 5b8825231c2156e2962b11d5a6e82b9a84dea840